doc/src/site/sphinx/advanced.rst

```diff
@@ -0,0 +1,40 @@
+Advanced Documentation
+**********************
+
+Zookeeper
+=========
+
+One of the necessities of Sparkta is to keep track of all policies running and their status. Since Zookeeper is a
+Sparkta dependency and a common piece in any Big Data architecture we save all this metadata in Zookeeper nodes.
+
+All this info is saved under the znode `/stratio/sparkta`. Under this root we save input and output fragments,
+policies and policy statuses.
+
+Fragments are the JSON blocks of and input or an output. We use them for reusing them when managing policies with the
+user interface. An example of an input fragment::
+
```

Comment:
Do you think that could be better this: 

> "Fragments are JSON blocks of inputs/outputs that will be included in a policy. If one fragment is changed, all policies that include it will be automated changed too. In fact, it is a nice way to reuse inputs / outputs between policies." 

instead of

> "Fragments are the JSON blocks of and input or an output. We use them for reusing them when managing policies with the user interface".